lib/pf/UnifiedApi.pm

```diff
@@ -203,6 +203,16 @@ our @API_V1_ROUTES = (
             },
             subroutes => undef,
         },
+    },
+    {
+        controller => 'Redis',
+        collection => {
+            subroutes    => {
+                'queue/stats' => {
+                    get => 'queue'
+                },
+            },
+        },
```

Comment:
To avoid the resource end points from being created add 

resource => undef,

Reply:
added in https://github.com/inverse-inc/packetfence/pull/3158/commits/369898600038d89e2b35cc4e1e854b1fdab9b5ae